README.md

```diff
@@ -1,182 +1,317 @@
 # Ruby Holidays Gem [![Build Status](https://travis-ci.org/holidays/holidays.svg?branch=master)](https://travis-ci.org/holidays/holidays)
 
-A set of functions to deal with holidays in Ruby.
-
-Extends Ruby's built-in Date class and supports custom holiday definition lists.
+Functionality to deal with holidays in Ruby.
```

Comment:
I think that in this case original was better. It clearly described what is included in this library. New version is shorter, but original was not so long to make shortening it necessary.

Reply:
Thank you! I've taken this into account and pushed up a new commit (37a2575). I'm going to go ahead and merge to clear this off of my plate but if you have additional feedback please add it here, I am more than happy to continue iterating so this is as clear as can be.